addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -167,7 +169,23 @@
                             <field name="active" invisible="1"/>
                             <field name="mailing_type" widget="radio" options="{'horizontal': true}" invisible="1"
                                 attrs="{'readonly': [('state', '!=', 'draft')]}" force_save="1"/>
-                            <field class="o_text_overflow" name="subject" string="Subject" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}" widget="char_emojis" placeholder="e.g. New Sale on all T-shirts"/>
+                            <label for="subject">Subject</label>
+                            <div class="o_mass_mailing_subject d-flex flex-row align-items-baseline">
+                                <field class="o_text_overflow" name="subject" string="Subject"
+                                    attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"
+                                    widget="char_emojis" placeholder="e.g. New Sale on all T-shirts"/>
+                                <field name="favorite" invisible="1"/>
+                                <button type="object" name="action_add_favorite"
+                                    class="o_mass_mailing_favorite p-0"
+                                    icon="fa-star-o"
+                                    attrs="{'invisible': [('favorite', '=', True)]}"
+                                    title="Add to templates"/>
```

Comment:
Note that we talk about favorites in enterprise, I suppose we should use the same wording ?

Reply:
I'll use "template" on the enterprise PR :)